internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,74 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	if len(data) == 0 {
+		err := errors.New("no data")
+		log.Println("error:", err)
+		return 0, 0, err
+	}
+	// str - слайс строк где str[0]-кол-во шагов ,str[1]-длительность тренировки; разделитель по знаку "запятой"
+	str := strings.Split(data, ",")
+	if len(str) != 2 {
+		err := errors.New("want - steps,duration; but have ")
+		log.Println("error:", err, data)
+		return 0, 0, err
+	}
+	// steps преобразованное интовое значение кол-во шагов
+	steps, err := strconv.Atoi(str[0])
+	if err != nil {
+		err := errors.New("want string to parse into int ; but have ")
+		log.Println("error:", err, str[0])
+		return 0, 0, err
```

Comment:
1. Теряешь текст самой ошибки преобразования
2. Если ошибку возвращаем, то лучше не выводить, вызывающая сторона сама решит, что делать с ошибкой
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```